catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/metacard/workspace/transformer/impl/EmbeddedListMetacardsHandlerTest.java

```diff
@@ -62,7 +69,13 @@
   public void setup() throws MalformedURLException {
     embeddedListMetacardsHandler = new EmbeddedListMetacardsHandler(actionRegistry);
     final Action action = getMockAction();
-    doReturn(singletonList(action)).when(actionRegistry).list(any());
+    doReturn(singletonList(action))
+        .when(actionRegistry)
+        .list(
```

Comment:
I'm seeing an intermittent compilation error here:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:testCompile (default-testCompile) on project catalog-ui-search: Compilation failure
[ERROR] /projects/ddf/catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/metacard/workspace/transformer/impl/EmbeddedListMetacardsHandlerTest.java:[74,14] incompatible types: inferred type does not conform to upper bound(s)
[ERROR]     inferred: java.util.Map<? extends java.lang.String,? extends ddf.catalog.data.Metacard>
[ERROR]     upper bound(s): java.util.Map<? extends java.lang.String,? extends capture#1 of ? extends ddf.catalog.data.Metacard>,java.util.Map<? extends java.lang.String,? extends ddf.catalog.data.Metacard>,java.lang.Object
```

Reply:
Qualifying `allOf` with `Matchers.<Map<String, Metacard>>` appears to have fixed it.